content/guides/content-management/allowed-child-types.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Allowed child types"
+metaTitle: "admin ui - allowed child types"
+metaDescription: "sensenet allowed child types"
+---
+
+# Allowed chiled types
+
+In a file system you can store any type (file or folder) anywhere, whereas in the sensenet content repository with the help of allowed child type it is possible to define restrictions on what content types the different containers can contain.
+You can configure allowed child types in the content type definition (CTD) of the different types. For example a Calendar can only contain Events, a Document Library can only contain Folders and Files, etc. These settings can be overridden on the specific content, for example you can modify any of your document libraries to contain images too. There are also some special types that behave differently: a Folder for example can never define child types, it will always inherit its parent settings. A SystemFolder will allow every type by default and can be created anywhere in the repository.
```

Comment:
```suggestion
You can configure allowed child types in the content type definition (CTD) of the different types. For example a *Calendar* can only contain *Events*, a *Document Library* can only contain *Folders* and *Files*, etc. These settings can be overridden on the specific content, for example you can modify any of your document libraries to contain images too. There are also some special types that behave differently: a *Folder* for example can never define child types, it will always inherit its parent settings. A *SystemFolder* will allow every type by default and can be created anywhere in the repository.
```